[OCPDOCS#19136]: CQA 2.0 configuring arbiter node (#115237)
openshift/openshift-docs: modules/understanding-agent-install.adoc

* **A single-node {product-title} cluster**: A node that is both a control plane and compute.
* **A three-node {product-title} cluster** : A compact cluster that has three control plane nodes that are also compute nodes.
* **Highly available {product-title} cluster (HA)**: Three control plane nodes with any number of compute nodes.

> * **An {product-title} cluster with local arbiter node**: Two control plane nodes with one local arbiter node. For more information, see the 'About local arbiter node' section.

Author: I understand that in the ticket you had suggested moving this section below "Platform ‘none’ DNS requirements”, but I added this here as it seemed more relevant to me. Please let me know if this looks good here. 
If it doesn't make sense here, we can discuss a more precise location for this.

> * **An {product-title} cluster with local arbiter node**: Two control plane nodes with one local arbiter node. For more information, see the 'About local arbiter node' section.

Reviewer: Interesting, I thought there was already a little blurb here for two node openshift. I would change three things though: Use double quotes when referencing the section, delete the word "section", and slightly rename the text in bold to better match the actual TNO docs:

```suggestion
* **Two-Node {product-title} cluster with Arbiter**: Two control plane nodes with one local arbiter node. For more information, see "Local arbiter node configuration".
```

(I also updated the section name based on my other comment, but just make sure it matches whatever the actual module name is)
